package.json

```diff
@@ -18,6 +18,7 @@
     "publish-storybook": "bash .scripts/publish_storybook.sh"
   },
   "devDependencies": {
+    "@kadira/storybook": "^1.35.1",
```

Comment:
I dont' see any other reference to storybook in package.json. We had npm install storybook -S to get the project to run.